assets/style/style.css

```diff
@@ -0,0 +1,276 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 2. v2.0 | 20110126
+  License: none (public domain)
+*/
+
+a,
+abbr,
+acronym,
+address,
+applet,
+article,
+aside,
+audio,
+b,
+big,
+blockquote,
+body,
+canvas,
+caption,
+center,
+cite,
+code,
+dd,
+del,
+details,
+dfn,
+div,
+dl,
+dt,
+em,
+embed,
+fieldset,
+figcaption,
+figure,
+footer,
+form,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+header,
+hgroup,
+html,
+i,
+iframe,
+img,
+ins,
+kbd,
+label,
+legend,
+li,
+mark,
+menu,
+nav,
+object,
+ol,
+output,
+p,
+pre,
+q,
+ruby,
+s,
+samp,
+section,
+small,
+span,
+strike,
+strong,
+sub,
+summary,
+sup,
+table,
+tbody,
+td,
+tfoot,
+th,
+thead,
+time,
+tr,
+tt,
+u,
+ul,
+var,
+video {
+  border: 0;
+  font-size: 100%;
+  margin: 0;
+  padding: 0;
+  vertical-align: baseline;
+}
+
+* {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.main-container {
+  display: grid;
+  grid-template-rows: repeat(2, auto);
+  justify-content: center;
+  margin-top: 20px;
+}
+
+.img-div {
+  background-image: url('../img/your_verse_hero.png');
+  background-position: 50% 100%;
+  background-repeat: no-repeat;
+  border-bottom: 2px solid #fff;
+  border-left: 2px solid #747474;
+  border-right: 2px solid #747474;
+  border-top: 2px solid #747474;
+  height: 545px;
+  width: 1024px;
+}
+
+.sec-layer {
+  border-bottom: 2px solid #747474;
+  border-left: 2px solid #747474;
+  border-right: 2px solid #747474;
+  display: grid;
+  grid-template-rows: repeat(2, auto);
+}
+
+.sec-img-div {
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+}
+
+.under-img {
+  border-right: 2px solid #fff;
+  height: 180px;
+  width: 255px;
+}
+
+.footer-part {
+  background-color: #fff;
+  display: grid;
+  grid-template-rows: repeat(2, auto);
+  padding: 20px;
+  width: 1020px;
+}
+
+.footer-up {
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+}
+
+.top-footer-list {
+  display: grid;
+  grid-template-columns: repeat(8, auto);
+  justify-content: center;
+}
+
+.nav-img {
+  height: 30px;
+  width: 30px;
+}
+
+.list-con {
+  display: grid;
+  grid-template-columns: repeat(8, auto);
+  height: 40px;
+  justify-content: center;
+  justify-items: center;
+}
+
+.nav-btn {
+  background: linear-gradient(to top, #4e4c4c 50%, #5f5d5d 50%);
+  background-color: #545252;
```

Comment:
This `background-color` is redundant. It has no visual effect because of the `linear-gradient`.